sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -91,32 +95,88 @@ credential.update("{new_api_key}");
 
 ## Key concepts
 ### Client
-The Form Recognizer client library provides a [FormRecognizerClient][form_recognizer_sync_client] and 
-[FormRecognizerAsyncClient][form_recognizer_async_client]. It provides both synchronous and
-asynchronous operations to access a specific use of Form Recognizer, such as training custom models,
-extracting layout and receipt information from documents.
+The Form Recognizer client library provides a [FormRecognizerClient][form_recognizer_sync_client], 
+[FormRecognizerAsyncClient][form_recognizer_async_client] and [FormTrainingClient][form_training_sync_client],
+[FormTrainingAsyncClient][form_training_async_client]. It also supports model management operations to
```

Comment:
Confused why you're highlighting only model management scenarios here, I think you don't need to include it